content/en/docs/ambient/usage/waypoint/index.md

```diff
@@ -107,10 +107,14 @@ The `--for` parameter to `istioctl waypoint apply` can be used to change the [tr
 | `waypoint-for` value | Traffic type |
```

Comment:
```suggestion
| `waypoint-for` value |  Destination type |
```

Reply:
actually more relevant than traffic vs destination here might be adding "original" or something along those lines. Maybe `Original Destination Type`